lib/rubocop/cop/rspec/empty_hook.rb

```diff
@@ -33,15 +34,11 @@ class EmptyHook < Cop
 
         def on_block(node)
           empty_hook?(node) do |hook|
-            add_offense(hook)
-          end
-        end
-
-        def autocorrect(node)
-          lambda do |corrector|
-            block = node.parent
-            range = range_with_surrounding_space(range: block.loc.expression)
-            corrector.remove(range)
+            add_offense(hook) do |corrector|
+              block = hook.parent
```

Comment:
block is actually just the `node`, so we can remove this line and use node directly